utils/githooks/pre-commit.d/30-yamllint.sh

```diff
@@ -14,4 +15,13 @@ then
 fi
 
 echo "Checking yaml formatting"
-yamllint --strict .
+targets=(
+    '*.yml'
+    '*.yaml'
+)
+
+files=${files:-$(git diff --diff-filter=ACMRTUXB --name-only --cached -- "${targets[@]}")}
```

Comment:
This is going to be a common construct.  I should have looked through the other hooks to notice that.  This could probably be DRYed out into a common function, maybe taking the `diff-filter` as an argument.

Although, this whole thing (lines 23-27) is probably better written as:

```sh
git diff --cached --name-only --diff-filter=ACMRTUXB -z -- "${targets[@]}" | xargs -r0 yamllint --strict
```

That eliminates the fragility of not quoting the arguments, etc.